db/src/models/Tag.js

```diff
@@ -12,10 +12,13 @@ export default class Tag extends Model {
     },
   }
 
+  static sanitizeValue = (value) => value.toLowerCase().replace('#', '').replace(',', '');
```

Comment:
this is actually a great example/use case of something that we could put into the nina-lib if we use this kind of pattern. if it's a one-off don't worry about it, but many moons ago i had a c++ header/module that handle these kinds of common string replacement functions.